lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -23,6 +23,9 @@ def self.provision_execute(_params = {}, _secrets = {}, context = {})
           return BuiltinRunnner.error!({}, :cause => "Unable to find MiqReqeustTask id: [#{object_id}]")                        if miq_request_task.nil?
           return BuiltinRunnner.error!({}, :cause => "Calling provision_execute on non-provisioning request: [#{object_type}]") unless miq_request_task.class < ::MiqProvision
 
+          miq_request_task.options_will_change!
+          miq_request_task.options.merge!(context.input.symbolize_keys)
```

Comment:
Not for this PR, but I wonder if we should only merge in keys that already exist?  That is, if the incoming input has extra stuff, we probably don't want to merge that in?  Not sure - would like to discuss first.